watertap/unit_models/tests/test_uv_aop.py

```diff
@@ -234,5 +236,29 @@ def test_solution(self, UV_frame):
             m.fs.unit.reaction_rate_constant["Liq", "NDMA"]
         )
         assert pytest.approx(1829.525, rel=1e-3) == value(
-            m.fs.unit.electricity_demand_phase_comp[0, "Liq", "NDMA"]
+            m.fs.unit.electricity_demand[0]
+        )
+
+    @pytest.mark.requires_idaes_solver
+    @pytest.mark.component
+    def test_costing(self, UV_frame):
+        m = UV_frame
+
+        m.fs.costing = WaterTAPCosting()
+        m.fs.costing.base_currency = pyunits.USD_2020
+
+        m.fs.unit.costing = UnitModelCostingBlock(
+            default={
+                "flowsheet_costing_block": m.fs.costing,
+            },
+        )
+        m.fs.costing.cost_process()
+        results = solver.solve(m)
+
+        assert results.solver.termination_condition == TerminationCondition.optimal
+
+        # Check solutions
+        assert pytest.approx(967.423, rel=1e-5) == value(m.fs.unit.costing.capital_cost)
```

Comment:
At 1 kg/s of feed flow, is this reasonable cost and is there any need to also test an industrial scale?